src/wp-includes/option.php

```diff
@@ -2210,14 +2163,14 @@ function update_network_option( $network_id, $option, $value ) {
 			 * As NULL is unique in the database, skip checking an old or new value of NULL
 			 * against any other value.
 			 */
-			( ! is_multisite() || ! ( null === $raw_old_value || null === $value ) ) &&
-			_is_equal_database_value( $raw_old_value, $value )
+			( ! is_multisite() || ! ( null === $old_value || null === $value ) ) &&
+			_is_equal_database_value( $old_value, $value )
 		)
 	) {
 		return false;
 	}
 
-	if ( $default_value === $raw_old_value ) {
+	if ( $default_value === $old_value ) {
```

Comment:
Couldn't verify but this will introduce error in Yoast. Per https://core.trac.wordpress.org/ticket/59360#comment:35